examples/src/fungible-token.js

```diff
@@ -1,87 +1,136 @@
-import {
-  NearBindgen,
-  call,
-  view,
-  initialize,
-  near,
-  LookupMap,
-  assert,
-} from "near-sdk-js";
+import { NearBindgen, call, view, initialize, near, LookupMap, assert } from "near-sdk-js";
 
 @NearBindgen({ initRequired: true })
 export class FungibleToken {
-  constructor() {
-    this.accounts = new LookupMap("a");
-    this.totalSupply = 0n;
-  }
+  accounts = new LookupMap("a");
+  accountRegistrants = new LookupMap("r");
+  accountDeposits = new LookupMap("c");
+  totalSupply = "0";
```

Comment:
Any reason to change it to string?

Reply:
I was getting BigInt -> string errors in the `_serialize()` function within `/build` during deployment in tests. A line similar to this one: 

```js
  return JSON.stringify(value);
```

Reply:
BigInt serialization was added in this PR: https://github.com/near/near-sdk-js/pull/228
So it should work. Please, check it. If it will fail anyway - let's try to discuss and fix it here.

Reply:
maps save bigints now to state